Mocky/Templates/Mock.swifttemplate

```diff
@@ -216,10 +222,18 @@ class MethodWrapper {
         guard !method.isInitializer else { return "" }
         guard !method.returnTypeName.isVoid else { return "" }
         if method.parameters.isEmpty {
-            return "return methodReturnValue(.\(prototype)) as! \(method.returnTypeName) "
+            return "guard let value = methodReturnValue(.\(prototype)) as? \(method.returnTypeName) else {\n" +
+            "\t\t\tprint(\"[FATAL] stub return value not specified for \(method.name). Use given.\")\n" +
```

Comment:
could you please replace that print and faliture message with one method? 
something like:
```
func stubFatalError(methodName: Stirng) -> Never {
let failureMessage = "MESSAGE"
print(failureMessage)
fatalError(failureMessage)
}
```
of course make that method in library sources and make it global.

Reply:
good point - I will add that as AC for simplify library case.